src/vs/workbench/contrib/notebook/browser/contrib/find/notebookFind.ts

```diff
@@ -77,7 +78,12 @@ registerAction2(class extends Action2 {
 		}
 
 		const controller = editor.getContribution<NotebookFindContrib>(NotebookFindContrib.id);
-		controller.show();
+
```

Comment:
With this check, it automatically searches in selected cells by default whenever there is multi selection. I'm not sure yet if this should be a setting or not in the future.

Reply:
see: https://github.com/microsoft/vscode/issues/213433